webapp/crashstats/api/tests/test_views.py

```diff
@@ -110,7 +110,7 @@ def test_base_classes_raise_not_found(self):
     def test_option_CORS(self):
         """OPTIONS request for model_wrapper returns CORS headers"""
         url = reverse("api:model_wrapper", args=("NoOp",))
-        response = self.client.options(url, HTTP_ORIGIN="http://example.com")
+        response = self.client.options(url, headers={"origin": "http://example.com"})
```

Comment:
The Django test client takes a `headers` argument instead of headers as kwargs. I don't know which Django version changed this--it's possible it's been this way for a while and we never updated.